src/app/containers/Copyright/index.test.jsx

```diff
@@ -13,6 +13,7 @@ const newsServiceContextStub = {
   imageCopyrightOffscreenText: 'Image source, ',
 };
```

Comment:
Likewise for the test it'd be good to add `lang: 'en-GB'` here in the News Service context stub.